wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -4,12 +4,10 @@
 {% block bodyclass %}page-explorer {% if ordering == 'ord' %}reordering{% endif %}{% endblock %}
 
 {% block content %}
-    <header class="header merged no-border">
-        <h1 class="visuallyhidden">Explorer</h1>
-
-        {% breadcrumbs parent_page 'wagtailadmin_explore' url_root_name='wagtailadmin_explore_root' %}
-    </header>
-
+    {% page_permissions parent_page as page_perms %}
```

Comment:
Pretty sure `page_perms` is no longer needed - can maybe be removed. 
Also can probably be removed from the `breadcrumbs` template tag.

Reply:
I think page_perms are being used for move page button to appear. Not so sure if it'll work without it but I'll try out